File: Patterns in C/Pattern 20.cpp
# include <stdio.h>
int main()
{
	int rows,space;
	char ch='A';
	printf("Enter No of Rows ");
	scanf("%d",&rows);
	
	for(int i=rows; i>=1; i--)
	{
		for(space=0; space<rows-i; space++)
		{
			printf("  ");
		}
		for(int j=i; j<=2*i-1; j++)
		{
			printf("%c ",ch);
		}
		for(int j=0; j<i-1; j++)
		{
			printf("%c ",ch);
		}
		printf("\n");
		ch++;
	} 
	return 0;
}

File: Patterns in Java/Pattern_20.java
import java.util.Scanner;

public class Pattern_20 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int rows, space;
        char ch = 'A';
        System.out.print("Enter No of Rows: ");
        rows = sc.nextInt();

        for (int i = rows; i >= 1; i--) {
            for (space = 0; space < rows - i; space++) {
                System.out.print("  ");
            }
            for (int j = i; j <= 2 * i - 1; j++) {
                System.out.print(ch + " ");
            }
            for (int j = 0; j < i - 1; j++) {
                System.out.print(ch + " ");
            }
            System.out.println();
            ch++;
        }
    }
}